### PR TITLE
translation and some style

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: src/www/services_dyndns_edit.php [hunk 1]
```diff
@@ -291,141 +291,141 @@ function _onTypeChange(type){
...
-                    <td><a id="help_for_host" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Hostname");?></td>
+                    <td><a id="help_for_host" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("Hostname") ?></td>
                     <td>
-                      <input name="host" type="text" id="host" value="<?=$pconfig['host'];?>" />
+                      <input name="host" type="text" id="host" value="<?= $pconfig['host'] ?>" />
                       <div class="hidden" for="help_for_host">
-                        <span class="text-danger"><strong><?=gettext("Note:");?><br /></strong></span>
-                        <?=gettext("Enter the complete host/domain name.  example:  myhost.dyndns.org");?><br />
-                        <?=gettext("For he.net tunnelbroker, enter your tunnel ID");?>
+                        <span class="text-danger"><strong><?= gettext("Note:") ?><br /></strong></span>
```
Comment (fichtner) [inline src/www/services_dyndns_edit.php]: Please let us get rid of all the iffy usage of Note: Warning: Whatnot: and the weird markup thereof. Useful help message and help message only. :)

File: src/www/services_ntpd.php [hunk 2]
```diff
@@ -321,7 +321,7 @@ function removeRow() {
                         <input class="btn btn-default btn-xs" type="button" onclick="show_advanced('showstatisticsbox', 'showstatistics')" value="<?=gettext("Advanced");?>" /> - <?=gettext("Show statistics logging options");?>
                       </div>
                       <div id="showstatistics" style="display:none">
-                      <strong><?=gettext("Warning: ")?></strong><?=gettext("these options will create persistant daily log files in /var/log/ntp."); ?>
+                      <?= sprintf(gettext("%sWarning:%s these options will create persistant daily log files in /var/log/ntp."),'<strong>','</strong>') ?>
```
Comment (fichtner) [inline src/www/services_ntpd.php]: this warning marker too, also capital "T", also "persistent"

**Comment by fabianfrz** [comment]
@fichtner I removed the "Note" and the "Warning"

**Comment by fichtner** [comment]
merged, thank you!

**Pull request merged**